srcpkgs/ntl/update

```diff
@@ -0,0 +1 @@
+site="${homepage}/download.html"
```

Comment:
Maybe comment in template that homepage is used in update file too? So it isn't clobbered accidentally.

Reply:
Isn't `"${homepage}"` the default for `site` in any case?

Reply:
Right, but if homepage moves to a github website or whatever the path won't exist anymore.

Reply:
I changed site to be explicit and not use `${homepage}`, that's what seems to be done in most templates. I didn't see any notice in the template about syncing with `update`, but I think if the homepage changes for some reason and update-check can't find site anymore it will complain about "no version found" or something like that.